rust/README.axum.md

```diff
@@ -0,0 +1,566 @@
+<div align="center">
+  <a href="https://github.com/{{github-name}}/{{repo-name}}" target="_blank">
+    <img src="https://raw.githubusercontent.com/{{github-name}}/{{repo-name}}/main/assets/a_logo.png" alt="{{repo-name}} Logo" width="100"></img>
```

Comment:
Not sure why, but these ended up templating in the username for me twice. For the first one, I ended up with

```html
  <a href="https://github.com/bgins/bgins/test-axum-template" target="_blank">
```

Reply:
oh interesting, will gander.

Reply:
This was from entering `bgins/rust-template-test` instead of `rust-template` when prompted for the name of the repo.